File: scratch_logo.js
const https = require('https');

https.get({
  hostname: 'www.unidraw.com',
  path: '/index.php',
  rejectUnauthorized: false
}, (resp) => {
  let data = '';
  resp.on('data', (chunk) => { data += chunk; });
  resp.on('end', () => {
    const imgRegex = /<img[^>]+src="([^">]+)"/ig;
    let match;
    const imgs = [];
    while ((match = imgRegex.exec(data)) !== null) {
      imgs.push(match[1]);
    }
    console.log(imgs);
  });
}).on("error", (err) => {
  console.log("Error: " + err.message);
});

File: scratch_logo.py
import urllib.request
import re

try:
    req = urllib.request.Request(
        'https://www.unidraw.com/index.php',
        headers={'User-Agent': 'Mozilla/5.0'}
    )
    with urllib.request.urlopen(req) as response:
        html = response.read().decode('utf-8')
        logos = re.findall(r'src="([^"]*logo[^"]*)"', html, re.IGNORECASE)
        imgs = re.findall(r'<img[^>]+src="([^">]+)"', html, re.IGNORECASE)
        print("Logos found:", logos)
        print("All images found:", imgs)
except Exception as e:
    print(e)
